geomstats/geometry/matrices.py

```diff
@@ -67,40 +92,40 @@ def transpose(mat):
         -------
         transpose : array-like, shape=[n_samples, dim, dim]
         """
-        is_vec = (gs.ndim(gs.array(mat)) == 3)
-        axes = (0, 2, 1) if is_vec else (1, 0)
+        is_vectorized = (gs.ndim(gs.array(mat)) == 3)
+        axes = (0, 2, 1) if is_vectorized else (1, 0)
         return gs.transpose(mat, axes)
 
-    @staticmethod
-    def vector_from_matrix(matrix):
```

Comment:
I agree that this was not the most useful method, but is there a specific reason why you deleted it?

Reply:
Yes, this was guided by what we discussed with stefan about manifold classes handling only one kind of representation for points and tangent vectors...

I think it will be less misleading for users as maintainers, so that we never have to wonder what kind of shape a specific method expects

Reply:
There's plenty of similar code in other matrix classes, for instance in SPD

Reply:
there's even a `symmetric_matrix_from_vector` using for loops

Reply:
I also did a 'search:vector from matrix' accross the repo and such methods clearly don't seem to be used anywhere, apart from some tests. Personnally I would delete all of them so that we feel lighter :)

Reply:
ok, so after a check such conversion methods are all absent in 'matrices.py' and 'general_linear.py'. I suggest we deal with SPD in another PR later, this guy is bigger ^^

Reply:
and if we ever want those back it'll be easy to reimplement them properly vectorized, by calling `gs.reshape` with the correct `axes` argument passed

Reply:
Yes, agree, we can delete it, especially if it's not used, and this would be a good opportunity to rewrite it better later.

I think manifolds should have different representations, but with one unique default representation so that the users do not worry indeed.